devices/stm32h7b3.yaml

```diff
@@ -155,6 +155,14 @@ RTC:
     - fields/rtc/v3.yaml
     - collect/rtc/alarm.yaml
 
+
+SAI1:
```

Comment:
Still trying to wrap my head around how to review something like this...

Why is this only added for the h7b3 and not h7a3 and h7b0? Is the bug only in the svd for the h7b3? From what I can see they all have those peripherals, right?

Reply:
Interrupts is better to look in the end of "mmap": https://github.com/stm32-rs/stm32-rs-mmaps/blob/86e7310017ab9a8919d985c3580e75ff6e81ca2a/stm32h7a3.mmap

> Still trying to wrap my head around how to review something like this...

Do not overwork yourself. I've asked you first of all to test your write permissions in the repo.

Reply:
Thanks :)